### PR TITLE
Feature/collectd

### PR DESCRIPTION
**Pull request by keithchambers**
Collectd configured to gather metrics and statistics for Linux system components

### CODE REVIEW

File: roles/collectd/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- name: enable epel repository
+  sude: yes
```
Comment (stevendborrelli) [inline roles/collectd/tasks/main.yml]: Should be sudo....this causes the build to fail.

**Comment by keithchambers** [comment]
@stevendborrelli Fixed!  So awesome that we have automated PR testing now!!!  :+1: Thank you @cdmitri and team!!

**Comment by siddharthist** [comment]
I'm testing this now.

**Comment by siddharthist** [comment]
Results:
 * `/usr/sbin/collectd -C /etc/collectd.conf -f` now runs on the servers at boot
 * Mesos, Marathon, and Consul are all responding normally

**Pull request merged**